samples/DurableFunctions/LocalFunctionProj/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:7.0 AS installer-env
+
+COPY . /src/dotnet-function-app
+WORKDIR /src/dotnet-function-app
+RUN dotnet publish *.csproj --output /home/site/wwwroot
+
+# To enable ssh & remote debugging on app service change the base image to the one below
+# FROM mcr.microsoft.com/azure-functions/dotnet-isolated:3.0-dotnet-isolated7.0-appservice
+FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated7.0
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    AzureFunctionsJobHost__Logging__Console__IsEnabled=true
+
+COPY --from=installer-env ["/home/site/wwwroot", "/home/site/wwwroot"]
+ENV WEBSITE_HOSTNAME="localhost:80"
+ENV AzureWebJobsStorage=""
+ENV SQLDB_Connection=""
```

Comment:
Should we add some sort of comment reminding users to populate these two?

Reply:
Though perhaps they shouldn't do that in the Dockerfile, as that risks pushing the creds. Not sure what's the recommended approach here.